### PR TITLE
[hotfix] GHA: build 'develop' branch with correct mvn command

### PR DESCRIPTION
**Pull request by dizzzz**
the devil is in the details.

### CODE REVIEW

**Review by line-o — COMMENTED** [review]
If I understand correctly, with `-Ddependency-check.skip=true` changed dependencies are not picked up. Or did I misunderstand what this does?

**Comment by dizzzz** [comment]
AFAIK it skips the time consuming OWASP Dependency-Check ?

**Comment by line-o** [comment]
OK, then I will re-add this to my fast-exist-build alias ;)

**Review by line-o — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -33,4 +33,4 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-        run: mvn -B verify org.sonarsource.scanner.maven:sonar-maven-plugin:sonar
+        run: mvn -V -B -e -Ddependency-check.skip=true -Ddocker=false -P \!mac-dmg-on-mac,\!codesign-mac-dmg,\!mac-dmg-on-unix,\!installer,\!concurrency-stress-tests,\!micro-benchmarks,\!build-dist-archives,skip-build-dist-archives verify
```
Comment (line-o) [inline .github/workflows/build.yml:36]: `org.sonarsource.scanner.maven:sonar-maven-plugin:sonar` is not needed?

**Comment by dizzzz** [comment]
will be delivered with  #3669

**Pull request closed without merging**